src/containers/dashboard/tag-editor.svelte

```diff
@@ -1,19 +1,18 @@
 <script>
-  import { stores } from '@sapper/app';
-  import Card from 'ui/card.svelte';
-  import Button from 'ui/button.svelte';
-  import TextField from 'ui/text-field.svelte';
+  import { getContext } from 'svelte';
+  import { Card, Button, TextField } from 'attractions';
+  import { snackbarContextKey } from 'attractions/src/snackbar';
```

Comment:
Am I correct to assume that importing from `src` is one of the things you are working on in the library?

Reply:
Not yet, but this is something to overcome with Rollup

Reply:
Which I could use your assistance with. I've pushed all my local changes to Attractions as a bunch of little PRs, and after merging them, maybe you could set up Rollup for it